### ISSUE TITLE
Redirect the authorized user 

### ISSUE THREAD

**Issue by kalinkaaaa14** [BinaryStudioAcademy/bsa-2023-towhub#300]
- [ ] Business should be redirected to dashboard
- [ ] Customer should be redirected to home page
- [ ] Driver should be redirected to the screen with trucks selection on a shift

Check if header changes for authorized user
Decide options in burger menu for all types of customer with qa and add it

**Comment by Some14u** [comment]
- Business should be redirected to the dashboard.
Currently, it is working as expected.

- Customer should be redirected to the home page.
Currently, it is working as expected.

- Driver should be redirected to the screen with truck selection on a shift.
Currently, there is no redirection; this page does not exist. It is being handled by @mV22110702 , and the redirect is embedded in his pull request. When it is merged, it will work.

- Check if the header changes for an authorised user.
Yes, the header changes for each type of user. If you are referring to the burger menu, please see the next point. Otherwise, the header changes to display the user's name.

- Decide on options in the burger menu for all types of customers with QA and add them.
[Here is the link](https://bsa2023.slack.com/archives/C05P451QRSL/p1695219308091609) to the discussion. Five people participated in it. We have currently settled on the following options:

> For Business:
> Profile: Currently, such a page does not exist.
> Log-out: It will work when it is merged by @h0wter (no link to orders because they are accessible through the dashboard).
> 
> For Customer:
> Order history: It will appear when merged by @AlexVlaso.
> Profile: Currently, such a page does not exist.
> Log-out: It will work when merged by @h0wter.
> 
> For Driver:
> Order history: Currently, such a page does not exist (but the results of @AlexVlaso work can be adapted).
> Log-out: It will work when merged by @h0wter and @kalinkaaaa14.

In conclusion, there is no need to take action on this ticket because all the elements are either covered by not yet merged pull requests or already exist in their pull requests. I can compile a list of links that the team is using and insert them in advance, but I'm not sure if it's necessary, considering that this information is already available in their pull requests.

**Comment by Some14u** [comment]
This must be tested after all mentioned above successfully merged.
